source/databricks/settlement_report/settlement_report_job/domain/report_generator.py

```diff
@@ -17,6 +18,50 @@
 log = Logger(__name__)
 
 
+def execute(spark: SparkSession, args: SettlementReportArgs) -> None:
```

Comment:
How about a more specific name. Perhaps something like this?
```suggestion
def execute_assemble_report(spark: SparkSession, args: SettlementReportArgs) -> None:
```

Reply:
This is outdated